src/module.ts

```diff
@@ -746,6 +754,8 @@ interface ModulePublicRuntimeConfig {
 
   navigation: ModuleOptions['navigation']
 
+  enableContentHead: ModuleOptions['enableContentHead']
```

Comment:
Maybe `contentHeadMeta` or `contentMetaTags`, instead of `enableContentHead`. WDYT?

Reply:
Yes as a boolean, or maybe simpler `contentHead`

Reply:
Like `documentDriven` option, users can set it to false